components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2008,20 +2008,6 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
                 id: attributionId,
                 spendingLimit: this.defaultSpendingLimit,
             });
-
-            // For all team members that didn't explicitly choose yet where their usage should be attributed to,
-            // we simplify the UX by automatically attributing their usage to this recently-upgraded team.
-            // Note: This default choice can be changed at any time by members in their personal billing settings.
-            const members = await this.teamDB.findMembersByTeam(teamId);
-            await Promise.all(
-                members.map(async (m) => {
-                    const u = await this.userDB.findUserById(m.userId);
-                    if (u && !u.usageAttributionId) {
-                        u.usageAttributionId = attributionId;
-                        await this.userDB.storeUser(u);
-                    }
-                }),
-            );
```

Comment:
Why is this removed?

Reply:
The logic of this change is:
1. Stop setting/unsetting `User.usageAttributionId` automatically, but give full explicit control of this value to the user
2. When attributing instances on start, we already look at the explicit user attribution, but if none is set and attribution, or it's set to an invalid choice, we should either guess what the sensible non-ambiguous attribution should be, or throw an error in the backend and show a prompt in the front-end in order to ask the user what their explicit choice is

Step 1. was partly implemented by https://github.com/gitpod-io/gitpod/pull/11584, but this particular location was forgotten so I'm removing it now

Step 2. is to be implemented in this PR (the whole non-ambiguous logic, error case, and Modal shown to the user)